1.4.6: compilation error (clang 9.0.0) when debug is enabled

**Describe the bug**

> " lambda capture 'this' is not used"

```
c++ -O2 -pipe -flax-vector-conversions -fstack-protector-strong -fno-strict-aliasing   -O2 -pipe  -flax-vector-conversions -fstack-protector-strong -fno-strict-aliasing  -Wall -fPIE -fvisibility=hidden -fstack-protector -pthread  -DNDEBUG  -DZT_USE_X64_ASM_SALSA2012 -DZT_BUILD_PLATFORM=7 -DZT_BUILD_ARCHITECTURE=2 -DZT_SOFTWARE_UPDATE_DEFAULT="\"disable\"" -std=c++11  -O2 -pipe  -flax-vector-conversions -fstack-protector-strong -fno-strict-aliasing  -Wall -fPIE -fvisibility=hidden -fstack-protector -pthread  -DNDEBUG  -DZT_USE_X64_ASM_SALSA2012 -DZT_BUILD_PLATFORM=7 -DZT_BUILD_ARCHITECTURE=2 -DZT_SOFTWARE_UPDATE_DEFAULT="\"disable\"" -Wall -Werror -g -pthread   -DZT_TRACE -DZT_USE_X64_ASM_SALSA2012 -DZT_BUILD_PLATFORM=7 -DZT_BUILD_ARCHITECTURE=2 -DZT_SOFTWARE_UPDATE_DEFAULT="\"disable\"" -std=c++11    -c -o one.o one.cpp
controller/DBMirrorSet.cpp:39:18: error: lambda capture 'this' is not used [-Werror,-Wunused-lambda-capture]
                                (*db)->each([this,&dbs,&db](uint64_t networkId,const nlohmann::json &network,uint64_t memberId,const nlohmann::json &member) {
                                             ^~~~~
1 error generated.
gmake[2]: *** [<builtin>: controller/DBMirrorSet.o] Error 1
gmake[2]: *** Waiting for unfinished jobs....
In file included from controller/EmbeddedNetworkController.cpp:35:
controller/EmbeddedNetworkController.hpp:137:6: error: private field '_listenPort' is not used [-Werror,-Wunused-private-field]
        int _listenPort;
            ^
controller/EmbeddedNetworkController.hpp:154:12: error: private field '_mqc' is not used [-Werror,-Wunused-private-field]
        MQConfig *_mqc;
                  ^
2 errors generated.
gmake[2]: *** [<builtin>: controller/EmbeddedNetworkController.o] Error 1
gmake[2]: Leaving directory '/tmp/usr/ports/net/zerotier/work/ZeroTierOne-1.4.6'
gmake[1]: *** [make-bsd.mk:163: debug] Error 2
gmake[1]: Leaving directory '/tmp/usr/ports/net/zerotier/work/ZeroTierOne-1.4.6'
===> Compilation failed unexpectedly.
Try to set MAKE_JOBS_UNSAFE=yes and rebuild before reporting the failure to
the maintainer.
*** Error code 1
```

**To Reproduce**
Steps to reproduce the behavior:

Compile net/zerotier 1.4.6 under FreeBSD ports building framework with debug option enabled. If debug is off, zt compiles Just Fine.

**Expected behavior**

Builds with perfect fluffy clouds.

**Screenshots**

**Desktop (please complete the following information):**

- FreeBSD 12.1Rp1 amd64, FreeBSD 13.0-CURRENT amd64
- zt 1.4.6
- clang 9.0.0

--- a/make-bsd.mk
+++ b/make-bsd.mk
@@ -13,7 +13,7 @@ ifeq ($(ZT_SANITIZE),1)
 endif
 # "make debug" is a shortcut for this
 ifeq ($(ZT_DEBUG),1)
-	CFLAGS+=-Wall -Werror -g -pthread $(INCLUDES) $(DEFS)
+	CFLAGS+=-Wall -g -pthread $(INCLUDES) $(DEFS)
 	LDFLAGS+=
 	STRIP=echo
 	ZT_TRACE=1

